docs/content/docs/testing/telemetry.md

```diff
@@ -13,21 +13,21 @@ your other telemetry, and route the event data in whatever way you need.
 ## Collect Fluid Framework logs with a custom `ITelemetryBaseLogger`
 
 The `ITelemetryBaseLogger` is an interface within the `@fluidframework/common-definitions` package. This interface can
-be implemented and passed into the client's `createContainer()` and `getContainer()` methods via the config parameter.
+be implemented and passed into the client's constructor via the props parameter.
 
-All Fluid service clients (for example, `AzureContainerConfig` and `TinyliciousContainerConfig`) allow passing a `logger?:
-ITelemetryBaseLogger` into the service client configuration. Both `createContainer()` and `getContainer()` methods will
-create an instance of a `Loader` class object, where the logger defined in the service client configuration is passed in
+All Fluid service clients (for example, `AzureClientProps` and `TinyliciousClientProps`) allow passing a `logger?: ITelemetryBaseLogger`
+into the service client props. Both `createContainer()` and `getContainer()` methods will
+create an instance of a `Loader` class object, where the logger defined in the service client props is passed in
 as an optional parameter, `ILoaderProps.logger`, in the `Loader` constructor argument.
 
-[TinyliciousContainerConfig](https://github.com/microsoft/FluidFramework/blob/main/experimental/framework/tinylicious-client/src/interfaces.ts#L9)
-interface definition takes an optional parameter `logger`. (The definition is similar to `AzureContainerConfig` interface)
+[TinyliciousClientProps](https://github.com/microsoft/FluidFramework/blob/main/packages/framework/tinylicious-client/src/interfaces.ts#L17)
+interface definition takes an optional parameter `logger`. (The definition is similar to `AzureClientProps` interface)
 
-[createContainer()](https://github.com/microsoft/FluidFramework/blob/main/experimental/framework/tinylicious-client/src/TinyliciousClient.ts#L50)
-interface definition takes a `TinyliciousContainerConfig` type as its `serviceContainerConfig` argument. (The
+[createContainer()](https://github.com/microsoft/FluidFramework/blob/main/packages/framework/tinylicious-client/src/TinyliciousClient.ts#L56)
+interface definition takes a `TinyliciousClientProps` type as its `serviceContainerConfig` argument. (The
 `AzureClient` will take `AzureContainerConfig` for its respective methods)
 
-[getContainer()](https://github.com/microsoft/FluidFramework/blob/main/experimental/framework/tinylicious-client/src/TinyliciousClient.ts#L65)
+[getContainer()](https://github.com/microsoft/FluidFramework/blob/main/packages/framework/tinylicious-client/src/TinyliciousClient.ts#L72)
 interface definition takes a `TinyliciousContainerConfig` type as its `serviceContainerConfig` argument. (The
```

Comment:
```suggestion
interface definition takes the `id` of the container and its `containerSchema`. (The
```

Reply:
marking it as resolved since `getContainer` is no longer relevant and taken out